src/containers/Header/index.js

```diff
@@ -38,38 +41,17 @@ class Header extends Component {
 
   handleClick = event => {
     const mode = event.currentTarget.getAttribute('value');
-    if (mode !== MODE) {
-      switch (mode) {
-        case MAINNET:
-          analytics(ANALYTIC_TYPES.event, {
-            eventCategory: 'mode switch',
-            eventAction: MAINNET_LINK,
-          });
-          window.location = MAINNET_LINK;
-          break;
-        case TESTNET:
-          analytics(ANALYTIC_TYPES.event, {
-            eventCategory: 'mode switch',
-            eventAction: TESTNET_LINK,
-          });
-          window.location = TESTNET_LINK;
-          break;
-        case DEVNET:
-          analytics(ANALYTIC_TYPES.event, {
-            eventCategory: 'mode switch',
-            eventAction: DEVNET_LINK,
-          });
-          window.location = DEVNET_LINK;
-          break;
-        default:
-          analytics(ANALYTIC_TYPES.event, {
-            eventCategory: 'mode switch',
-            eventAction: MAINNET_LINK,
-          });
-          window.location = MAINNET_LINK;
-          break;
-      }
+
+    if (mode === MODE) {
+      return;
     }
+
+    const desiredLink = ENV_LINK_MAP[mode] ?? process.env.REACT_APP_MAINNET_LINK;
```

Comment:
Should this output a log if the desired link doesn't match anything? That seems like an important thing to know about when setting up an environment.

Reply:
Verdict - yep that probably makes sense, but is beyond the scope of this ticket